addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -2739,6 +2739,7 @@ export class Wysiwyg extends Component {
         }
         const closestDialog = e.target.closest('.o_dialog, .o_web_editor_dialog');
         if (
+            e.target.closest("#oe_snippets") ||
```

Comment:
This can be ignored in stable IMO because the SnippetsMenu is inside the iframe and this code will only be executed if the iframe has been blurred, which should not occur when clicking within it.

In more details, here's the chain of events that this code is handling:
- Click outside the iframe
- If click is inside a dialog, on the toolbar, or now, in the SnippetsMenu
	- Delay the "blur" event to the next click which is not one of these elements or to the next blur event
- If not, and a blur has been delay, commit the changes
- If not and a blur has not been delayed, [the blur event on the iframe will commit the changes](https://github.com/odoo/odoo/blob/a65051a650cf24e9260dad5ebf4b7df4eec4702d/addons/web_editor/static/src/js/wysiwyg/wysiwyg.js#L2761)

If we remove the code I added, the blur event will commit the changes while we're interacting with the SnippetsMenu for the first time, which will reset the SnippetsMenu.

It is not necessary in stable because the SnippetsMenu is inside the iframe, so a click within it will not trigger a blur event.

However, in stable, weird things happen which I assume are wanted, when clicking in the chatter for example, the SnippetsMenu will be restarted and it looks like a glitch. Maybe something could be better there 🤷🏻

Reply:
> This can be ignored in stable IMO because the SnippetsMenu is inside the iframe

@detrouxdev That's exactly the point: the website SnippetsMenu is not in the iframe, so if this is necessary for the mass mailing now that the menu is out of the iframe, why was it not needed before for the website? In any case, **huge** lack of comments :grimacing: